qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -219,94 +219,16 @@ admin_daemon()
     ceph --admin-daemon ${asok_file} $@
 }
 
-status()
```

Comment:
It is useful (at least for me) when testing manually to check the current state, but sure I can live without it (especially if it adds additional work to maintain the script).